dbt/adapters/spark/impl.py

```diff
@@ -132,38 +134,79 @@ def add_schema_to_cache(self, schema) -> str:
         # so jinja doesn't render things
         return ""
 
+    def parse_information(self, table_name):
```

Comment:
This should probably return a dictionary `{"info_type": info_value" for info_type, info_value, _ in table_results if not info_type.startswith("#")}`. Instead it's either an empty list or a string that looks like a dictionary / json. Let's keep it as one data type.

Reply:
Great catch!

It looks like it is a `str` on the relation itself:
```python
@dataclass(frozen=True, eq=False, repr=False)
class SparkRelation(BaseRelation):
...
    information: Optional[str] = None
```
How about keeping it a string?

Reply:
I love this and we can (I checked, it wouldn't break out plugins). But, it requires some debugging outside the scope of this PR. Comment left instead